argo-event/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,23 @@
+# Used to 
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Add a flag to toggle
{{-if .values.monitor_enabled }}

Reply:
Why we need to give the flexibility to the user to create their own podmonitor rather than have a fix podmonitor template and just allow them to enable or disable it. Because I think this podmonitor is used to monitor the event source, eventbus and sensor, the user not need to modify the detail since it already support monitor event source, eventbus and sensor